arns/arns-token-1.md

```diff
@@ -10,6 +10,7 @@
 | ------- | --------------------------------------------------------- | ---------- |
 | 1.0.0   | Initial version of the **ARNS-TOKEN-1** specification.    | 2024-09-01 |
 | 1.0.1   | Fixed Credit/Debit response notices for Transfer handler. | 2024-09-24 |
+| 1.0.2   | Added 'description' and 'keywords' metadata.              | 2024-10-14 |
```

Comment:
recommend 1.1.0 - adding new apis is a feature.

Reply:
Great point.  Updated this to 1.1